### PR TITLE
Wrappers for drupal behavior and plain jQeury created.

### PR DESCRIPTION
**Pull request by NotAccessible**
js:wrap tasks writen.
js:components task updated.
Gulpfile updated.

### CODE REVIEW

File: _dev/assets/js/behavior-wrap.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  Drupal.behaviors.fesk = {
+    attach: function (context, settings) {
+
+      <!-- inject:jQuery -->
+      <!-- endinject -->
+
+    }
+  };
+
+}(jQuery, Drupal));
```
Comment (el22or) [inline _dev/assets/js/behavior-wrap.js]: Wrong end of the line.

File: _dev/assets/js/jquery-wrap.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+$(function() {
+
+  <!-- inject:jQuery -->
+  <!-- endinject -->
+
+});
```
Comment (el22or) [inline _dev/assets/js/jquery-wrap.js]: Wrong end of the line.

File: _dev/assets/js/jquery-wrap.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+$(function() {
```
Comment (el22or) [inline _dev/assets/js/jquery-wrap.js]: Shouldn't this be written as `(function($) {`?
And, the end of the scope as `})(jQuery)`?

File: _dev/gulp/settings/files.js [hunk 4]
```diff
@@ -31,7 +31,10 @@ module.exports = {
   tmp: {
     css: [settings.path.tmp.css + '/**/*.css'],
     sass: [settings.path.tmp.sass + '/**/*.sass'],
-    js: [settings.path.tmp.js + '/**/*.js'],
+    js: {
```
Comment (el22or) [inline _dev/gulp/settings/files.js:34]: I think there will be issue with this.
`files` meant to be _minimatch_ pattern style for searching files. `path` is the one which contains paths as object map.

File: _dev/gulp/tasks/js/components.js [hunk 5]
```diff
@@ -1,17 +1,23 @@
...
+      var source = settings.files.tmp.js.static;
+    }
+
+    return gulp.src(source)
       .pipe(plugins.plumber({errorHandler: handlers.error}))
       // .pipe(plugins.sourcemaps.init({loadMaps: true}))
       .pipe(plugins.uglify())
       // .pipe(plugins.sourcemaps.write('./'))
-      .pipe(gulp.dest(settings.path.theme.templates));
+      .pipe(gulp.dest(settings.path.theme.js));
```
Comment (el22or) [inline _dev/gulp/tasks/js/components.js]: Again, `components` are files from Pattern Lab components. The idea was to place them together with TWIG and CSS files, inside `templates` folder in Drupal. Then, you manually create `library` for those components and point onto CSS and JS files. They should NOT be combined together with other files, nor moved to `assets` folder.

File: _dev/gulp/tasks/js/components.js [hunk 6]
```diff
@@ -1,17 +1,23 @@
...
   });
-};
+};
```
Comment (el22or) [inline _dev/gulp/tasks/js/components.js]: Wrong end of the line.

File: _dev/gulp/tasks/js/wrap.js [hunk 7]
```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+var inject = require("gulp-inject");
+var foreach = require('gulp-foreach');
+var rename = require('gulp-rename');
+var path = require('path');
+
+module.exports = function (gulp, plugins, settings, handlers) {
+
+  gulp.task('js:wrap', function () {
```
Comment (el22or) [inline _dev/gulp/tasks/js/wrap.js:10]: Can you please explain this task? I do not understand what exactly is doing?

**Review by el22or — COMMENTED** [review]
First of all, I like how you got into fesk! Like it a lot!!!
Second, I do not understand this pull request completely, so I think we should talk.

**Comment by el22or** [comment]
Zasto si zatvorio ovo? :)
Ne treba ti vise?

**Comment by NotAccessible** [comment]
Napravio sam novi pa sam ovaj zatvorio. Nisam znao kako da dodam izmene ovom PR-u.

**Pull request closed without merging**